modules/logging/variables.tf

```diff
@@ -53,3 +53,9 @@ variable "enable_cluster_logging" {
   type        = bool
   default     = false
 }
+
+variable "kibana_subdomain_name" {
```

Comment:
for whatever reason, they removed the default for some reason in [0.22.0] vs [0.21.0]


[0.21.0]: https://github.com/cloudposse/terraform-aws-elasticsearch/blob/0.21.0/variables.tf#L266-L270
[0.22.0]: https://github.com/cloudposse/terraform-aws-elasticsearch/blob/0.22.0/variables.tf#L212-L215